src/parquet/file/metadata.cc

```diff
@@ -485,16 +485,17 @@ class ColumnChunkMetaDataBuilder::ColumnChunkMetaDataBuilderImpl {
     column_chunk_->meta_data.__set_total_uncompressed_size(uncompressed_size);
     column_chunk_->meta_data.__set_total_compressed_size(compressed_size);
     std::vector<format::Encoding::type> thrift_encodings;
-    thrift_encodings.push_back(ToThrift(Encoding::RLE));
     if (has_dictionary) {
-      thrift_encodings.push_back(ToThrift(properties_->dictionary_page_encoding()));
-      // add the encoding only if it is unique
-      if (properties_->version() == ParquetVersion::PARQUET_2_0) {
-        thrift_encodings.push_back(ToThrift(properties_->dictionary_index_encoding()));
+      thrift_encodings.push_back(ToThrift(properties_->dictionary_index_encoding()));
+      if (properties_->version() == ParquetVersion::PARQUET_1_0) {
```

Comment:
I don't think we need this check anymore. `properties_->dictionary_page_encoding()` also check it ?
I think the spec says: https://github.com/Parquet/parquet-format/blob/master/src/thrift/parquet.thrift
PARQUET_1_0 : data (index) page: PLAIN_DICTIONARY, dictionary page: PLAIN_DICTIONARY
PARQUET_2_0 : data (index) page: RLE_DICTIONARY, dictionary page: PLAIN